static/docs/commands-reference/status.md

```diff
@@ -50,21 +50,45 @@ detected, `dvc status` prints this message:
 ```
 
 This says that no differences were detected, and therefore that no stages would
-be run again if `dvc repro` were executed.
+be run again by `dvc repro`.
 
 If instead, differences are detected, `dvc status` lists those changes. For each
-DVC-file (stage) with differences, the _dependencies_ and/or _outputs_ that
-differ are listed. For each item listed, either the file name or the checksum is
-shown, and additionally a status word is shown describing the change:
+DVC-file (stage) with differences, the changes in _dependencies_ and/or
+_outputs_ that differ are listed. For each item listed, either the file name or
+the checksum is shown, and additionally a status word is shown describing the
+changes in checksum or status. Given list provides a reference to both, the
+status of the DVC-file and differences in the DVC-files:
```

Comment:
`differences in the DVC-files` -> `and changes to individual dependencies and outputs described by the file`

`status of the DVC-file` -> `status of a DVC-file`

Reply:
Here, full suggestion for the last 2 sentences:

"For each item listed, either the file name or the checksum is shown, and additionally a status word is shown describing the changes. This changes list provides a reference to both the status of a DVC-file, as well as the changes to individual dependencies and outputs described in it:"

Reply:
revised the para based on these suggestions, kindly check them.

Reply:
Yes, thanks. But actually we should also end the paragraph in period `.` please (since there are columns `:` in the main bullets below. (Like I mentioned in https://github.com/iterative/dvc.org/pull/416#discussion_r307958841)